src/applications/ivc-champva/10-7959f-1/tests/e2e/10-7959f-1-FMP.cypress.spec.js

```diff
@@ -25,26 +31,7 @@ const testConfig = createTestConfig(
             .click();
         });
       },
-      'applicant-information': ({ afterHook }) => {
-        cy.injectAxeThenAxeCheck();
-        afterHook(() => {
-          cy.get('@testData').then(data => {
-            // eslint-disable-next-line no-console
-            console.log('data', data);
-            fillDateWebComponentPattern(
-              'veteransFullName',
-              data.veteranFullName,
-            );
-            fillDateWebComponentPattern(
-              'veteranDateOfBirth',
-              data.veteranDateOfBirth,
-            );
-            cy.axeCheck();
-            cy.findByText(/continue/i, { selector: 'button' }).click();
-          });
-        });
-      },
-      'identification-information': ({ afterHook }) => {
```

Comment:
Unnecessary (this page is automatically handled successfully so we don't have to explicitly state how the data should be filled, also the path was outdated)